### PR TITLE
added recipe for google-cloud-datastore

### PR DESCRIPTION
**Pull request by birnbaum**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by trendelkampschroer** [comment]
You can list me as a maintainer.

**Comment by mathiaseitz** [comment]
You can list me as a maintainer.

File: recipes/google-cloud-datastore/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (chrisburr) [inline recipes/google-cloud-datastore/meta.yaml]: Can you change this with the string in the example

File: recipes/google-cloud-datastore/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+{% set name = "google-cloud-datastore" %}
+{% set version = "1.8.0" %}
+{% set sha256 = "1bc0c6a9af953b71f0956faf775f000076c5631ce32a23f29a89986a84e3bf8b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/google-cloud-datastore/meta.yaml]: This isn't needed
```suggestion
```

File: recipes/google-cloud-datastore/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+test:
+  imports:
+    - google
+    - google.cloud
+    - google.cloud.datastore
+
+about:
+  home: https://github.com/GoogleCloudPlatform/google-cloud-python
+  license: Apache 2.0
```
Comment (chrisburr) [inline recipes/google-cloud-datastore/meta.yaml]: It's a tiny thing but the SPDX naming scheme puts a dash here:
```suggestion
  license: Apache-2.0
```

**Comment by birnbaum** [comment]
Thanks for your comments, I resolved the remaining issues!

**Pull request merged**